client/shipping-label/state/reducer.js

```diff
@@ -422,21 +422,24 @@ reducers[ SET_PACKAGE_TYPE ] = ( state, { packageId, boxTypeId } ) => {
 	const newPackages = { ...state.form.packages.selected };
```

Comment:
@dmsnell no solid reason yet - it fits in with some other patterns I've noted before. It may be the case that a different structure isn't needed after all, but it's worth a look.